llvm/tools/sycl-post-link/SYCLDeviceRequirements.cpp

```diff
@@ -40,12 +40,10 @@ void llvm::getSYCLDeviceRequirements(
   // Scan the module and if the metadata is present fill the corresponing
   // property with metadata's aspects
   constexpr std::pair<const char *, const char *> ReqdMDs[] = {
-      {"sycl_used_aspects", "aspects"},
-      {"sycl_fixed_targets", "fixed_target"},
-      {"reqd_work_group_size", "reqd_work_group_size"}};
+      {"sycl_used_aspects", "aspects"}, {"sycl_fixed_targets", "fixed_target"}};
 
   for (const auto &[MDName, MappedName] : ReqdMDs) {
-    std::set<uint32_t> Values;
+    std::set<int64_t> Values;
```

Comment:
Why is this change needed? Both `sycl_used_aspects` and `sycl_fixed_targets` are "internal" values of the implementation, they are not user-provided. And I don't think that any of them has to be 64-bit

Reply:
I think it might have been necessary before merging with #10614; now that that PRs changes are integrated with this PR it should be unnecessary.